source/PlayServicesResolver/src/SettingsDialog.cs

```diff
@@ -55,6 +56,7 @@ private class Settings {
                 verboseLogging = SettingsDialog.VerboseLogging;
                 autoResolutionDisabledWarning = SettingsDialog.AutoResolutionDisabledWarning;
                 promptBeforeAutoResolution = SettingsDialog.PromptBeforeAutoResolution;
+                autoResolutionDelay = Math.Min(Math.Max(0,SettingsDialog.AutoResolutionDelay),30);
```

Comment:
I'm curious why this would be limited to 30 seconds?

Reply:
I just think it should be limit to not let people shoot themselves in the foot by putting 9999+ delay time. 30 sec should be reasonable delay for popup if we don't stay away from keyboard

Reply:
Ok, worth moving the "30" into a constant so that it's self documenting?

e.g MAXIMUM_AUTO_RESOLVE_DELAY_TIME